packages/fluentui/react-northstar/src/components/Table/TableRow.tsx

```diff
@@ -44,6 +44,14 @@ export interface TableRowProps extends UIComponentProps {
    * Whether a row is currently selected or not.
    */
   selected?: boolean;
+
+  /**
+   * Called on click.
+   *
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props.
+   */
+  onClick?: ComponentEventHandler<TableRowProps>;
```

Comment:
In the propTypes too. That's actually from where we are generating the handledProps.